### ISSUE TITLE
Does bundle require security component to be installed?

### ISSUE THREAD

**Issue by jpedroh** [thecodingmachine/graphqlite-bundle#72]
Hello,

I've installed this in a Symfony Flex application that does not use the security bundle, for some other reasons.

However, there seems to be a dependency on security component inside the bundle code, as I'm having an error that says:
`"Could not autoload class 'Symfony\\Component\\Security\\Core\\User\\UserInterface' defined in @Type annotation of class 'TheCodingMachine\\Graphqlite\\Bundle\\Types\\SymfonyUserInterfaceType'"`

I've also followed configuration and have explicitly disabled security mutations, using:
```
graphqlite:
  security:
    enable_login: off
    enable_me: off
```

As stated in docs. However, notice here that, according to them, it shouldn't even be necessary to do this, as it says:
> By default, GraphQLite will enable "login" and "logout" mutations and the "me" query if the following conditions are met:
> - the "security" bundle is installed and configured (with a security provider and encoder)

Which is not my situation. I've also tried removing these lines, but also didn't have any success. I've tried installing symfony security bundle, and it works smoothly, however, ideally, I wouldn't want to have it installed. Any ideas of what could be done?

**Comment by moufmouf** [comment]
Hi @jpedroh,

Interesting!
I realize the unit tests always run with the security bundle installed, so indeed, there might be a hidden dependency.
I'll check this.

**Comment by moufmouf** [comment]
Should be fixed and tagged in 4.0.8!

**Comment by jpedroh** [comment]
Working sweet now. Thanks :D
